modules/unsupported/csv/src/test/java/org/geotools/data/csv/parse/CSVSpecifiedWKTStrategyTest.java

```diff
@@ -0,0 +1,73 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
```

Comment:
We will return this to public domain license, you asked for permission on geoserver devel to backport to GeoTools. A PMC member gave you permission etc...

Should also probably be (c) 2010-2013 OpenPlans as CSVDataStore was not yet written in 2001.